### PR TITLE
[bn] Localizes data/i18n/bn/bn.toml

### PR DESCRIPTION
**Pull request by nerdynikhil**
Localizes `data/i18n/bn/bn.toml` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

File: data/i18n/bn/bn.toml [hunk 1]
```diff
@@ -60,307 +60,307 @@
...
-#other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the "
+#other = " ডকুমেন্টেশন আর সক্রিয়ভাবে রক্ষণাবেক্ষণ করা হয় না। আপনি বর্তমানে যে সংস্করণটি দেখছেন সেটি একটি স্ট্যাটিক স্ন্যাপশট। আপ-টু-ডেট ডকুমেন্টেশনের জন্য, দেখুন "
 
 #[deprecation_file_warning]
-#other = "Deprecated"
+#other = "ডেপ্রিক্যাটেড"
 
 #[dockershim_message]
-#other = """Dockershim has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
+#other = """1.24 রিলিজ হিসাবে Dockershim কুবারনেটস প্রকল্প থেকে সরানো হয়েছে। বিস্তারি তথ্যের জন্য <a href="/dockershim">Dockershim Removal FAQ</a> পড়ুন। """
```
Comment (mitul3737) [inline data/i18n/bn/bn.toml:72]:
```suggestion
#other = """1.24 রিলিজ হিসাবে Dockershim কুবারনেটিস প্রকল্প থেকে সরানো হয়েছে। বিস্তারি তথ্যের জন্য <a href="/dockershim">Dockershim Removal FAQ</a> পড়ুন। """
```

File: data/i18n/bn/bn.toml [hunk 2]
```diff
@@ -374,56 +374,56 @@
...
-#other = "Subscribe"
+#other = "সাবস্ক্রাইব"
 
 #[synopsis_heading]
-#other = "Synopsis"
+#other = "সারমর্ম"
 
 #[thirdparty_message]
-#other = """This section links to third party projects that provide functionality required by Kubernetes. The Kubernetes project authors aren't responsible for these projects, which are listed alphabetically. To add a project to this list, read the <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> before submitting a change. <a href="#third-party-content-disclaimer">More information.</a>"""
+#other = """এই বিভাগটি তৃতীয় পক্ষের প্রকল্পগুলির সাথে লিঙ্ক করে যা Kubernetes দ্বারা প্রয়োজনীয় কার্যকারিতা প্রদান করে। Kubernetes প্রকল্প লেখক এই প্রকল্পগুলির জন্য দায়ী নয়, যা বর্ণানুক্রমিকভাবে তালিকাভুক্ত করা হয়েছে। এই তালিকায় একটি প্রকল্প যোগ করতে, একটি পরিবর্তন জমা দেওয়ার আগে <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> পড়ুন। <a href="#third-party-content-disclaimer">অধিক তথ্য</a>"""
```
Comment (mitul3737) [inline data/i18n/bn/bn.toml:398]:
```suggestion
#other = """এই বিভাগটি তৃতীয় পক্ষের (third party) প্রকল্পগুলির সাথে লিঙ্ক করে যা Kubernetes দ্বারা প্রয়োজনীয় কার্যকারিতা প্রদান করে। Kubernetes প্রকল্প লেখক এই প্রকল্পগুলির জন্য দায়ী নয়, যা বর্ণানুক্রমিকভাবে তালিকাভুক্ত করা হয়েছে। এই তালিকায় একটি প্রকল্প যোগ করতে, একটি পরিবর্তন জমা দেওয়ার আগে <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> পড়ুন। <a href="#third-party-content-disclaimer">অধিক তথ্য</a>"""
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@sftim Can you have a look into this ? Let me know if it looks ok 😊

**Comment by nerdynikhil** [comment]
thanks @mitul3737 for the help.

File: data/i18n/bn/bn.toml [hunk 3]
```diff
@@ -8,16 +8,16 @@
 #other = """<p>This page is automatically generated.</p><p>If you plan to report an issue with this page, mention that the page is auto-generated in your issue description. The fix may need to happen elsewhere in the Kubernetes project.</p>"""
 
 #[caution]
-#other = "Caution:"
+#other = "সতর্কতা:"
```
Comment (seokho-son) [inline data/i18n/bn/bn.toml:11]:
```suggestion
other = "সতর্কতা:"
```

> [caution]
> other = "Caution:"

`#` should be removed :) 
Please update all sections.

File: data/i18n/bn/bn.toml [hunk 4]
```diff
@@ -8,16 +8,16 @@
 #other = """<p>This page is automatically generated.</p><p>If you plan to report an issue with this page, mention that the page is auto-generated in your issue description. The fix may need to happen elsewhere in the Kubernetes project.</p>"""
```
Comment (seokho-son) [inline data/i18n/bn/bn.toml:8]: 4~8 lines are not localized.

**Review by seokho-son — COMMENTED** [review]
Hi @nerdynikhil @mitul3737  :)

You need to remove `#` for the keys and values to enable them. 
Also there are some keys which doesn't have localized value. (line 4 ~ 8)

You can think https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml is the template and we need to modify the value part only for the data/i18n/bn/bn.toml.

Please check the comments. :)

File: data/i18n/bn/bn.toml (hunk 4 above)
Comment (nerdynikhil) [reply data/i18n/bn/bn.toml]: thanks @seokho-son

**Review by mitul3737 — APPROVED** [review]

**Comment by mitul3737** [comment]
Great work @nerdynikhil 😊

**Pull request merged**